pyproject.toml

```diff
@@ -56,14 +56,11 @@ packages = ["sphinxlint"]
 [tool.hatch.version.raw-options]
 local_scheme = "no-local-version"
 
-[tool.black]
-
-[tool.ruff]
+[tool.ruff.lint]
 select = [
   "E", # pycodestyle errors
   "F", # pyflakes errors
   "I", # isort
-  "ISC", # flake8-implicit-str-concat
```

Comment:
I removed this since there was a CI warning saying that it might conflict with the formatting hook